openhtf/util/units/__init__.py

```diff
@@ -0,0 +1,1170 @@
+# coding: iso-8859-1
```

Comment:
I would actually prefer to have this generated as part of the build process rather than having it checked in.  Having it checked in means having the same information duplicated in two places, leaving opportunity for it to get out of sync.